kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_incorrect_workbasket_key.bpmn

```diff
@@ -4,7 +4,7 @@
     <bpmn:startEvent id="StartEvent_1">
       <bpmn:outgoing>SequenceFlow_1eiligh</bpmn:outgoing>
     </bpmn:startEvent>
-    <bpmn:serviceTask id="ServiceTask_0vg4xui" name="TaskWithIncorrectWorkbasketKey" camunda:class="pro.taskana.impl.InvalidWorkbasketKeySetter">
+    <bpmn:serviceTask id="ServiceTask_0vg4xui" name="TaskWithIncorrectWorkbasketKey" camunda:class="impl.io.kadai.InvalidWorkbasketKeySetter">
```

Comment:
io.kadai.impl.InvalidWorkbasketKeySetter ?

Reply:
You're right, I will change it